.travis.yml

```diff
@@ -16,8 +16,12 @@ matrix:
       env: TOXENV=py35
     - python: 3.6
       env: TOXENV=py36
-    - python: 3.7-dev
+    - python: 3.7
       env: TOXENV=py37
+      # begin: workaround to enable support for py37:
+      sudo: required
```

Comment:
[Travis are now recommending removing the __sudo__ tag](https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration).

"_If you currently specify __sudo: false__ in your __.travis.yml__, we recommend removing that configuration_"

Reply:
I think I addressed that in my PR which is hopefully going to be merged soon

Reply:
AFAIK, sudo was needed in order to install the newer py37. If that is not needed anymore we can drop it.

Travis recommented not using sudo from day one but reality is that quite often I faced myself in cases where sudo was mandatory for testing some stuff.

Reply:
Travis CI sunsetted their container-based infra. Everyone gets VMs now. So this option is basically garbage, therefore.

Reply:
Ouch.... not sure what to say about that. While I am fully aware that there are jobs that may not work as docker containers, they count probably for les that 0.1%. I guess it translates to "we failed to manage a production cluster". I guess it goes along the line with https://www.reddit.com/r/programming/comments/8avdrl/travis_ci_dropped_production_database_by/ :)